valhalla/baldr/pathlocation.h

```diff
@@ -215,6 +213,12 @@ struct PathLocation : public Location {
     if (loc.has_way_id()) {
       l.way_id_ = loc.way_id();
     }
+    if (loc.has_search_cutoff()) {
+      l.search_cutoff_ = loc.search_cutoff();
+    }
+    if (loc.has_street_side_tolerance()) {
+      l.street_side_tolerance_ = loc.street_side_tolerance();
+    }
```

Comment:
this is the important bit when moving from pbf to custom classes. tests coming shortly